scholia/app/templates/chemical.html

```diff
@@ -98,7 +132,9 @@
 SELECT ?date ?work ?workLabel ?type ?topics
 WITH {
   SELECT DISTINCT ?work WHERE {
-    {
+    {INCLUDE %RESULTS
```

Comment:
This line and the two next seems to be wrong.

Reply:
oh, should have spotted that :( Sorry!